lib/matplotlib/projections/geo.py

```diff
@@ -55,6 +56,10 @@ def cla(self):
         Axes.set_xlim(self, -np.pi, np.pi)
         Axes.set_ylim(self, -np.pi / 2.0, np.pi / 2.0)
 
+    def cla(self):
+        """Clear the current axes."""
+        self.clear()
```

Comment:
I think the `cla()` here can be inherited from the baseclass? So make sure that `Axes` has defined `cla()` to call `self.clear()`, which will then call your redefined `self.clear()` above. I'm pretty sure you already have this, so removing this method should be fine.